.github/workflows/build-vcpkg.yml

```diff
@@ -121,21 +109,87 @@ jobs:
         with:
           name: tfs-${{ matrix.name }}-${{ matrix.buildtype }}-luajit=${{ matrix.luajit }}-${{ github.sha }}
           path: ${{ runner.workspace }}/build/tfs
-        if: "! contains( matrix.os, 'windows')"
 
-      - name: Upload artifact binary (exe)
+      - name: Prepare datapack contents
+        run: find . -maxdepth 1 ! -name data ! -name config.lua.dist ! -name key.pem ! -name LICENSE ! -name README.md ! -name schema.sql -exec rm -r {} \;
+        shell: bash
```

Comment:
What does this new step do?

Reply:
The diff checker is trolling you, if you view the raw file, the step above uploads the binary to the artifact, 
this step cleans up the runner workspace from unnecessary files such as build files, and retails data folder and config etc, 
and the step below uploads these remaining files to the artifact

See artifacts from my trigger test:
<https://github.com/Znote/forgottenserver/actions/runs/1076419470#artifacts>

Reply:
Yeah the differ strikes again